interceptors/Mementifier.cfc

```diff
@@ -160,11 +160,21 @@ component {
 			"trustedGetters"   : isNull( this.memento.trustedGetters ) ? variables.$mementifierSettings.trustedGetters : this.memento.trustedGetters
 		};
 		// Param arguments according to instance > settings chain precedence
-		param arguments.trustedGetters   = thisMemento.trustedGetters;
-		param arguments.iso8601Format    = thisMemento.iso8601Format;
-		param arguments.dateMask         = thisMemento.dateMask;
-		param arguments.timeMask         = thisMemento.timeMask;
-		param arguments.autoCastBooleans = thisMemento.autoCastBooleans;
+		if(isNull(arguments.trustedGetters)){
```

Comment:
So the `param` is not respecting full null support?

Reply:
why do you use param inside a function, I mean generally we use param for form and URL data.

Reply:
`param` is a generic language construct that tests for a parameter's existence, tests its data type, and, if
a default value is not assigned, optionally provides one.  A parameter can be any valid scope variable.  Thus, it can be used not only for the form/url scopes, but for any scope.  That's why it should test If the paramed variable is null and assign it the default value.

Reply:
If we enable full null support and pass any one argument then remain argument is initiated with a null value and not set a default value, Because we define it in function, the param works fine for undefined variables/arguments.

Reply:
Can you please let me know how I can reproduce this please.

Reply:
For reproduction first enabled full null support in an application using **this.nullSupport = true**;
Then just call getMemento for any object with **ignoreDefaults: true** 
_i.e. userBean.getMemento( includes: "name", excludes:"id",ignoreDefaults: true);_

Reply:
What CFML Engine are you using?

Reply:
I am using Lucee 5.3.9.141

Reply:
Thanks.  let me go and test.